tests/readme.md

```diff
@@ -86,3 +93,6 @@ Options include:
 - `PLUGINS_TEST_CLI_PATH` specifies an alternative path to a trunk binary
 - `PLUGINS_TEST_LINTER_VERSION` specifies a linter version semantic (KnownGoodVersion | Latest |
   version)
+- `PLUGINS_TEST_NEW_SNAPSHOT` if "true" tells tests to use an exact match of the linter version when
```

Comment:
```suggestion
- `PLUGINS_TEST_UPDATE_SNAPSHOTS` if "true" tells tests to use an exact match of the linter version when
```

Reply:
For future: If PLUGINS_TEST_LINTER_VERSION is version, add todo for make sure only run with one test

Reply:
Also todo: callout -u/our behavior for the flag